src/components/socials/index.js

```diff
@@ -2,20 +2,31 @@ import React from 'react'
 import socials from './content'
 import './socials.less'
 
-const Socials = () => (
-  <div className="socials">
-    {socials.map((social, i) => (
-      <a
-        className="socials__link"
-        href={social.link}
-        target="_blank"
-        rel="nofollow noopener noreferrer"
-        key={`socialsSlide${i}`}
-      >
-        <img className="socials__icon" src={social.icon} />
-      </a>
-    ))}
-  </div>
-)
+import onscroll from './onscroll'
 
-export default Socials
+class SocialsSmart extends React.Component {
+  componentDidMount() {
+    onscroll()
+  }
+
+  render() {
+    const className = `${this.props.sidebar ? 'socials__sidebar' : ''}`
```

Comment:
According to BEM. Double underscope is used only for child elements. For this context you should use a modificator: 
socials_sidebar

Reply:
fixed